src/main/java/hudson/plugins/jacoco/model/CoverageObject.java

```diff
@@ -203,7 +204,7 @@ public Coverage getLineCoverage() {
 	/**
 	 * Gets the build object that owns the whole coverage report tree.
 	 */
-	public abstract AbstractBuild<?,?> getBuild();
+	public abstract Run<?,?> getBuild();
```

Comment:
the other way to do it is to deprecate this and introduce a new method `getRun()`  let `getBuild()` attempt to cast the Run to a AbstractBuild and throw an exception if its not possible - which should be ok as the code that currently calls this would only call it for a non workflow job...